exercises/concept/calculator-conundrum/.docs/hints.md

```diff
@@ -1,2 +1,13 @@
 # Hints
 
+## General
+
+- [Exceptions][exceptions]: basic introduction to exceptions.
+- [Using exceptions][using-exceptions]: how to use exceptions.
+- [Exception handling][exception-handling]: how to catch exceptions.
```

Comment:
It would be great if the hints could be linked to individual tasks. Take for example [task 2](https://github.com/exercism/csharp/blob/2eaeb0a8c8b38e0cff3f762bd85d68d6c727cacb/exercises/concept/calculator-conundrum/.docs/instructions.md#2-handle-illegal-operations), which requires one to throw exceptions. Instead of having the throwing of exceptions as a general hint, you could move it to a task-specific hint:

```markdown
## 2. Handle illegal operations

- You need to [throw an exception][creating-and-throwing-exceptions] here.

The same goes for catching exceptions, which belongs to task 3.
```